zulip_bots/zulip_bots/bots/jira/jira.conf

```diff
@@ -2,3 +2,4 @@
 username = <your Jira email address>
 password = <your Jira password>
 domain = <your Jira domain>
+# domain_url = [optional] <your front facing Jira URL to use in links>
```

Comment:
Missing new line at the and of this file.

Reply:
You probably meant to write `display_url` here.

Reply:
Fixed